cover-icon-element.js

```diff
@@ -78,18 +81,20 @@ class CoverIconElement extends HTMLElement {
+CoverIconElement.dataImage1 = (color, opacity) => `data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='${encodeURIComponent(color)}' preserveAspectRatio='xMidYMid meet' focusable='false' viewBox='0 0 24 24'%3E%3Cg%3E%3Cpath d='M3 4H21V8H19V20H17V8H7V20H5V8H3V4'%3E%3C/path%3E%3Cpath d='M3 9H16V11H8V9' opacity='1'%3E%3C/path%3E%3Cpath d='M3 12H16V14H8V12' opacity='${encodeURIComponent(opacity)}'%3E%3C/path%3E%3Cpath d='M3 15H16V17H8V15' opacity='${encodeURIComponent(opacity)}'%3E%3C/path%3E%3Cpath d='M3 18H16V20H8V18' opacity='${encodeURIComponent(opacity)}'%3E%3C/path%3E%3C/g%3E%3C/svg%3E`;
 
-CoverIconElement.dataImage0 = (color) => `data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='${encodeURIComponent(color)}' preserveAspectRatio='xMidYMid meet' focusable='false' viewBox='0 0 24 24'%3E%3Cg%3E%3Cpath d='M3 4H21V8H19V20H17V8H7V20H5V8H3V4'%3E%3C/path%3E%3Cpath d='M3 9H16V11H8V9' opacity='0.3'%3E%3C/path%3E%3Cpath d='M3 12H16V14H8V12' opacity='0.3'%3E%3C/path%3E%3Cpath d='M3 15H16V17H8V15' opacity='0.3'%3E%3C/path%3E%3Cpath d='M3 18H16V20H8V18' opacity='0.3'%3E%3C/path%3E%3C/g%3E%3C/svg%3E`;
+CoverIconElement.dataImage0 = (color, opacity) => `data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='${encodeURIComponent(color)}' preserveAspectRatio='xMidYMid meet' focusable='false' viewBox='0 0 24 24'%3E%3Cg%3E%3Cpath d='M3 4H21V8H19V20H17V8H7V20H5V8H3V4'%3E%3C/path%3E%3Cpath d='M3 9H16V11H8V9' opacity='${encodeURIComponent(opacity)}'%3E%3C/path%3E%3Cpath d='M3 12H16V14H8V12' opacity='${encodeURIComponent(opacity)}'%3E%3C/path%3E%3Cpath d='M3 15H16V17H8V15' opacity='${encodeURIComponent(opacity)}'%3E%3C/path%3E%3Cpath d='M3 18H16V20H8V18' opacity='${encodeURIComponent(opacity)}'%3E%3C/path%3E%3C/g%3E%3C/svg%3E`;
 
 CoverIconElement.dataImageClosing = (color) => `data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='${encodeURIComponent(color)}' preserveAspectRatio='xMidYMid meet' focusable='false' viewBox='0 0 24 24'%3E%3Cg%3E%3Cpath d='M3 4H21V8H19V20H17V8H7V20H5V8H3V4'%3E%3C/path%3E%3Cpath d='M10 9 H14 V15 H16 L12 19 L 8 15 H10 Z' opacity='0.7'%3E%3C/path%3E%3C/g%3E%3C/svg%3E`;
 
 CoverIconElement.dataImageOpening = (color) => `data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='${encodeURIComponent(color)}' preserveAspectRatio='xMidYMid meet' focusable='false' viewBox='0 0 24 24'%3E%3Cg%3E%3Cpath d='M3 4H21V8H19V20H17V8H7V20H5V8H3V4'%3E%3C/path%3E%3Cpath d='M10 19 H14 V13 H16 L12 9 L8 13 H10 Z' opacity='0.7'%3E%3C/path%3E%3C/g%3E%3C/svg%3E`;
 
+CoverIconElement.dataImageError = (color) => `data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" role="img" width="1em" height="1em" preserveAspectRatio="xMidYMid meet" viewBox="0 0 24 24"><path d="M2 4h18v4h-2v12h-2V8H6v12H4V8H2V4m5 5h8v2H7V9m0 3h8v2H7v-2m13 7v-2h2v2h-2m0-4v-5h2v5h-2z" fill="${encodeURIComponent(color)}"/></svg>`;
```

Comment:
What do you think about changing this icon to have the exclamation mark "inside the window"? Similar to the closing and opening icons but with the exclamation arrow instead of the arrows.

I suggest this because adding the mark to the side, increases the width of the icon and may possibly mess up the icon positioning in very specific picture-elements use cases.

![error_icon_comparison](https://user-images.githubusercontent.com/1107697/141950651-fb443e84-7996-4659-b696-10f2f22599e2.gif)

Reply:
I have been lazy and just copy-pasted the standard material icon for the blind-alert, but obviously it can be done.